### PR TITLE
docs(react-color-picker): Updated documentation for the ColorPicker

### PR DESCRIPTION
**Pull request by ValentinaKozlova**
## New Behavior
- Fixed Spec
- Added MIGRATION.md
- Added info to the Readme.md

### CODE REVIEW

File: packages/react-components/react-color-picker-preview/library/README.md [hunk 1]
```diff
@@ -3,3 +3,47 @@
...
+
+The ColorPicker is used to browse through and select colors.
+By default, it lets people navigate through colors on a color spectrum; or specify a color in either Red-Green-Blue (RGB); or alpha color code; or Hexadecimal textboxes.
+
+## Usage
+
+To import React ColorPicker components:
+
+```tsx
+import { ColorPicker, ColorSwatch, ColorPickerOnSelectEventHandler } from '@fluentui/react-components';
```
Comment (dmytrokirpa) [inline packages/react-components/react-color-picker-preview/library/README.md:15]: nit

```suggestion
import { ColorPicker, ColorSwatch, type ColorPickerOnSelectEventHandler } from '@fluentui/react-components';
```

File: packages/react-components/react-color-picker-preview/library/README.md [hunk 2]
```diff
@@ -3,3 +3,47 @@
...
+```
+
+Simple example of ColorPicker Usage:
+
+```tsx
+import { tinycolor } from '@ctrl/tinycolor';
+import { ColorPicker, ColorSlider, AlphaSlider, ColorPickerProps, ColorArea } from '@fluentui/react-components';
```
Comment (dmytrokirpa) [inline packages/react-components/react-color-picker-preview/library/README.md:22]:
```suggestion
import { ColorPicker, ColorSlider, AlphaSlider, type ColorPickerProps, ColorArea } from '@fluentui/react-components';
```

File: packages/react-components/react-color-picker-preview/library/README.md [hunk 3]
```diff
@@ -3,3 +3,47 @@
 **React Color Picker components for [Fluent UI React](https://react.fluentui.dev/)**
 
 These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
+
+The ColorPicker is used to browse through and select colors.
+By default, it lets people navigate through colors on a color spectrum; or specify a color in either Red-Green-Blue (RGB); or alpha color code; or Hexadecimal textboxes.
```
Comment (dmytrokirpa) [inline packages/react-components/react-color-picker-preview/library/README.md:8]: Does the color picker currently support the RGB color format? My understanding was that it only supports HSV/HSL color formats.

**Review by dmytrokirpa — APPROVED** [review]

File: packages/react-components/react-color-picker-preview/library/README.md (hunk 3 above)
Comment (ValentinaKozlova) [reply packages/react-components/react-color-picker-preview/library/README.md:8]: Yes, ColorPicker itself works with HSV. But input boxes (which I'll add in stories) will support RGB and HEX. Here's an example of how it looks in Office:
image

Comment (ValentinaKozlova) [reply packages/react-components/react-color-picker-preview/library/README.md:8]: Though I think it's good to point out this behavior in the Spec. Thanks for catching that

**Pull request merged**